_posts/spells/first-level/2020-10-13-spells-1-absorb-elements.md

```diff
@@ -1,18 +1,18 @@
 ---
 title: Absorb Elements
-permalink: "/spells/cantrips/absorb-elements/"
+permalink: "/spells/first-level/absorb-elements/"
 layout: spell-post
 categories: [Spells, Cantrips]
-tags: [Artificer, Sorcerer, Wizard]
+tags: [Artificer, Sorcerer, Wizard, Druid, Ranger]
 source: "Basic Rules (PHB)"
-spell-level: Cantrip
-school: Conjuration
-casting-time: "1 Action"
-range: "60 feet"
-components: "V, S"
-duration: "Instantaneous"
+spell-level: "1st-level"
+school: Abjuration
+casting-time: "1 Reaction"
```

Comment:
1 reaction, which you take when you take acid, cold, fire, lightning or thunder damage